src/cli.js

```diff
@@ -63,12 +63,14 @@ const getDefaultAnswer = question => {
  *
  * @param {Object} args
  */
-const mainProcess = async ({ template, yes }) => {
+const mainProcess = async ({ template, yes, path }) => {
```

Comment:
To reduce complexity, I think this function should always take two arguments : 
- `templatePath` (Previously named template) that would either be the path to the default template (`../templates/default.md`) or the path to the user custom template (`path/to/my/custom/template`)
- `yes`

This means that before calling `mainProcess` we should have a function (like `getTemplatePath` for example) that would return `../templates/default.md` or `path/to/my/custom/template` if specified.

In addition, if custom template is not valid, I don't think that we should use default template, but show an error message telling the user that path is not correct and then abort process. 

What do you think ?

Reply:
At the first time reading the issue, I had a thought that you would use more options in the near future. But after reading this comment, I think it would be surely okay to simplify the logic in the current situation.

Thank you for the suggestion. I'm gonna have some look.

Reply:
I'm not saying that this Function will never take more than 2 args, just saying that in this case, having two args is enough 🙂